src/actions/ipfs.actions.ts

```diff
@@ -0,0 +1,32 @@
+import { logger } from "@blockr/blockr-logger";
+import * as fs from "fs";
+import { toast } from "react-toastify";
+import feedback from "../components/overview/feedback/feedback_mock/feedback.json";
+import { ApiService } from "../services/apiService";
+
+const apiService: ApiService = new ApiService();
+export const uploadPDFToIPFS = (base64ConvertedPDF: string) => {
+    apiService
+        .postDocumentToIPFS(base64ConvertedPDF)
+        .then((response) => {
+            toast.info("Uploaded document");
+            console.log(response);
```

Comment:
Best not to use console.log but the Blockr-Logger
```suggestion
            logger.info(response);
```